src/test/java/brave/opentracing/BraveSpanTest.java

```diff
@@ -136,11 +136,8 @@
     assertThat(spans).hasSize(3);
     assertThat(spans.get(0).traceId).isEqualTo(spans.get(1).traceId)
         .isEqualTo(spans.get(2).traceId);
-    assertThat(spans.get(1).id).isEqualTo(spans.get(2).id);
+    assertThat(spans.get(1).id).isNotEqualTo(spans.get(2).id);
     assertThat(spans.get(0).id).isNotEqualTo(spans.get(1).id);
-
-    assertThat(zipkin.spanStore().getDependencies(System.currentTimeMillis(), null))
```

Comment:
Oh, yeah, I'm running Zipkin 1.29.0 and it didn't have any issues representing the dependencies on my traces. I'll try to update the dependency here and see if the test passes with that assertion.